EpikCord/__init__.py

```diff
@@ -1923,6 +1932,118 @@ def unload_section(self, section: Union[CommandsSection, EventsSection]):
 #     def __init__(self, client: Client,data: dict):
 #         super().__init__(data)
 
+class VoiceUDPProtocol(asyncio.DatagramProtocol):...
+class VoiceWebsocketClient:
+    #This class shouldnt be used by users.Instead, it is given on voice channels
+    #This class, however, can be used by lavalink libraries
+    def __init__(self, client:Client, **kwargs):
+        self.client = client
+        self.guild_id = kwargs.get("guild_id")
+        self.channel_id = kwargs.get("channel_id")
+        self.HELLO = 8
+        self.IDENTIFY = 0
+        self.HEARTBEAT = 3
+        self.voice_connected =False
+        self.server_set = False
+        self.state_set = False
+        self.sequence = None
+        self.client.events["voice_state_update"] = self.voice_state_update
+        self.client.events["voice_server_update"] = self.voice_server_update
```

Comment:
Incorrect, `client.events` is the user-registered dict for events, it's just `client.event_name` and you are meant to register them in the `EventHandler` class